node/faas/faas.go

```diff
@@ -43,3 +51,36 @@ func kyc(tx *types.Transaction) error {
 	log.Println("KYC FaaS not implemented")
 	return nil
 }
+
+func elKYT(tx *types.Transaction) error {
```

Comment:
add comment pls for EL_KYT that the eigen stuff needs to be up and running + provide the link for the repo